### PR TITLE
Sldss 24653 create public api documentation for gdn url management service

### PR DESCRIPTION
**Pull request by achebanets**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+                      created: "2015-11-21T11:51:17Z"
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -H "Authorization: Bearer $smartlingToken" https://api.smartling.com/gdn-url-management-api/v2/projects/$smartlingProjectId/web-pages/details
+
+  /gdn-url-management-api/v2/projects/{projectId}/web-pages/{webPageUid}:
+    get:
+      description: >-
+        Get details for urls.
```
Comment (hchaps) [inline spec/swagger.yaml]: Retrieve detail for the URLs.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+      summary: Get details for urls
```
Comment (hchaps) [inline spec/swagger.yaml]: Get details for URLs

File: spec/swagger.yaml [hunk 3]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: projectId
+          in: path
+          required: true
+          type: string
+          description: Unique identifier of the project account.
```
Comment (hchaps) [inline spec/swagger.yaml]: (Same suggestion as previous one.)

File: spec/swagger.yaml [hunk 4]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+        - name: webPageUid
+          in: path
+          required: true
+          type: string
+          description: Unique identifier of the url.
```
Comment (hchaps) [inline spec/swagger.yaml]: Unique identifier of the URL.

File: spec/swagger.yaml [hunk 5]
```diff
@@ -7756,6 +7896,89 @@ definitions:
...
+  WebPageListRequest:
+    type: object
+    properties:
+      urlPath:
+        type: string
+        description: url
+        example: /home/page.htm
+      urlPathExactMatch:
+        type: boolean
+        description: Should apply exact match or not.
```
Comment (hchaps) [inline spec/swagger.yaml]: Should or should not apply exact match.

File: spec/swagger.yaml [hunk 6]
```diff
@@ -7756,6 +7896,89 @@ definitions:
...
+        example: false
+      domainName:
+        type: string
+        description: domain name
+        example: my.domain.com
+      urlCaptureDateAfter:
+        type: string
+        format: date-time
+        example: '2020-11-21T01:51:17Z'
+        description: The date after which the url was captured.
```
Comment (hchaps) [inline spec/swagger.yaml]: The date after which the URL was captured.

File: spec/swagger.yaml [hunk 7]
```diff
@@ -7756,6 +7896,89 @@ definitions:
...
+      urlCaptureDateBefore:
+        type: string
+        format: date-time
+        example: '2020-11-21T01:51:17Z'
+        description: The date by which the url was captured.
```
Comment (hchaps) [inline spec/swagger.yaml]: (Same comment as previous one.)

File: spec/swagger.yaml [hunk 8]
```diff
@@ -6125,6 +6125,146 @@ paths:
           schema:
             $ref: '#/definitions/SuccessResponse'
 
+  #
+  # GDN url management
+  #
+  /gdn-url-management-api/v2/projects/{projectId}/web-pages/list:
+    post:
+      description: >-
+        Get list of urls.
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml [hunk 9]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+      summary: Get list urls
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/swagger.yaml]: Changed to "Get list of URLs".

File: spec/swagger.yaml [hunk 10]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: projectId
+          in: path
+          required: true
+          type: string
+          description: Unique identifier of the project account.
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/swagger.yaml]: Done for all places (gun url management service)

File: spec/swagger.yaml [hunk 11]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+                      created: "2015-11-21T11:51:17Z"
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -H "Authorization: Bearer $smartlingToken" https://api.smartling.com/gdn-url-management-api/v2/projects/$smartlingProjectId/web-pages/list
+
+  /gdn-url-management-api/v2/projects/{projectId}/web-pages/details:
+    post:
+      description: >-
+        Get details for urls.
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml [hunk 12]
```diff
@@ -6125,6 +6125,146 @@ paths:
...
+      summary: Get details for urls
```
[the remark this answers is not in the document]
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 2 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 3 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 4 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 5 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 6 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 7 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

**Pull request merged**